components/gitpod-db/src/typeorm/migration/1621920691254-InitialComSetup.ts

```diff
@@ -1,27 +0,0 @@
-/**
```

Comment:
why was this removed?

Reply:
Please see the commit message:

>  [db] Remove unnecessary InitialComSetup migration (all tables already exist in Baseline)

This migration file is a no-op, because all tables it wants to create already existed before. @AlexTugarev and @geropl already agreed in principle. This is a minor drive-by fix, please ignore. 😇